buildscripts/buildRelease.sh

```diff
@@ -447,4 +462,51 @@ cd $WORKSPACE
 git checkout docs/app/data/footer-navigation.json
 git checkout docs/app/data/landing-page.json
 
+if [ "$BuildPackages" == "true" ]; then
+	# Loop while waiting for the $NextVersion-package-test branch to be merged into the Bower
+	# branch or deleted. Merging signals that the release is verified and so the NPM package may
+	# be created.
+    branchCheckDelay=30    
+    while :
+    do
+        branchExists=`git ls-remote --heads https://github.com/UXAspects/UXAspects.git $NextVersion-package-test | wc -l`
+        if [ $branchExists == 0 ] ; then
+		    latestBowerCommitID=`git rev-parse origin/bower`
+			echo latestBowerCommitID is $latestBowerCommitID
+			latestBowerCommitMessage=`git log --format=%s%b -n 1 $latestBowerCommitID`
+			echo latestBowerCommitMessage is $latestBowerCommitMessage
+			branchMerged=`echo "$latestBowerCommitMessage" | grep "$NextVersion-package-test" | wc -l`
+            # if this contains $NextVersion-package-test and $latestCommitID, the branch was merged
+            if [ $branchMerged == 1 ] ; then
+                echo Branch $NextVersion-package-test has been merged into the Bower branch. Create the NPM package.
+				echo
+				echo Creating the NPM package
+				if [ -d "$WORKSPACE/npm" ]; then
+					echo "Folder $WORKSPACE/npm exists... deleting it!"
+					rm -rf $WORKSPACE/npm
+				fi
+				mkdir -p $WORKSPACE/npm
+				cd $WORKSPACE/npm
+				cp -p -r $WORKSPACE/src/package.json .
+				cp -p -r $WORKSPACE/dist .
+				
+				# Copy the package to a folder on the Grid Hub machine.
+				echo Deleting old copy of NPM package on Selenium Grid Hub machine
+				ssh $SELENIUM_TEST_MACHINE_USER@$GridHubIPAddress rm -rf $REMOTE_NPM_PACKAGE_FOLDER
+
+				echo Copying NPM package to the Selenium Grid Hub machine
+				ssh $SELENIUM_TEST_MACHINE_USER@$GridHubIPAddress mkdir -p $REMOTE_NPM_PACKAGE_FOLDER
+				scp -r $WORKSPACE/npm/* $SELENIUM_TEST_MACHINE_USER@$GridHubIPAddress:$REMOTE_NPM_PACKAGE_FOLDER
```

Comment:
Publish with npm if possible. Try this: https://github.com/npm/npm/issues/9747#issuecomment-233714676